conf/battle/items.conf

```diff
@@ -79,13 +79,13 @@ autospell_stacking: no
 // Will disabled consumables (disabled by map_zone_db.conf) be consumed when trying to use them?
 // 1 (official): yes
 // 0: no
-item_restricted_consumption_type: 1
+item_restricted_consumption_type: yes
 
 // Enable all NPC to allow changing of equipments while interacting? (Note 1)
 // Script commands 'enable_items/disable_items' will not be override. (see doc/script_commands.txt)
 // 1 : yes(official)
 // 0 : no
-item_enabled_npc: 1
+item_enabled_npc: yes
```

Comment:
the comment on both lines 80~81 and 86~86 says that it uses a integer parameter, yet you changed it to 'yes', well my point is: edit the comment as well and get rid off of integers :P